src/sentry/api/endpoints/organization_events_stats.py

```diff
@@ -154,11 +154,19 @@ def get(self, request: Request, organization: Organization) -> Response:
                     )
                 )
             )
+
+            use_profiles = features.has(
```

Comment:
Could you add this to `self.get_features` so it gets batched? (lol sorry I don't have this on the /events/ api)

Reply:
this isnt on flagr so there's no benefit to doing this